docs/installation/configuration.rst

```diff
@@ -42,6 +42,25 @@ DATABASES
    `Django documentation on databases <https://docs.djangoproject.com/en/2
    .1/ref/settings/#databases>`_
 
+DEFAULT_FILE_STORAGE
+^^^^^^^^^^^^^^^^^^^^
+
+   By default, Pulp uses the local filesystem to store files. This can be configured though to
+   alternatively use `Amazon S3 <https://aws.amazon.com/s3/>`_. To use S3, set
+   ``DEFAULT_FILE_STORAGE`` to ``pulpcore.app.models.storage.S3Storage``. The default option which
```

Comment:
What about adding a sentence that identifies that the user needs to `pip install`?